monkey/infection_monkey/post_breach/actions/communicate_as_backdoor_user.py

```diff
@@ -79,28 +86,25 @@ def get_commandline_for_http_request(url, is_windows=is_windows_os()):
                 format_string = "wget -O/dev/null -q {url} --method=HEAD --timeout=10"
         return format_string.format(url=url)
 
-    def send_result_telemetry(self, exit_status, commandline, username):
+    def _get_result_for_telemetry(self, exit_status, commandline, username):
         """
-        Parses the result of the command and sends telemetry accordingly.
+        Parses the result of the command and returns it to be sent as telemetry from the master.
 
         :param exit_status: In both Windows and Linux, 0 exit code indicates success.
         :param commandline: Exact commandline which was executed, for reporting back.
         :param username: Username from which the command was executed, for reporting back.
         """
         if exit_status == 0:
-            PostBreachTelem(
-                self, (CREATED_PROCESS_AS_USER_SUCCESS_FORMAT.format(commandline, username), True)
-            ).send()
+            result = (CREATED_PROCESS_AS_USER_SUCCESS_FORMAT.format(commandline, username), True)
```

Comment:
I think it would be better to first translate the exit status to pba status (True/False) and then `_format_pba_status_string`. This is optional though, since making code cleaner is not the point of this refactoring